SPECS/cyrus-sasl-ldap/cyrus-sasl-ldap.spec

```diff
@@ -0,0 +1,214 @@
+%define _soversion 3
```

Comment:
Maybe let's add this comment for the future devs to explain what transpired here:

```suggestion
# NOTE: this is a copy of 'cyrus-sasl.spec' with the only difference being the `--with-ldap` config option meant to enable building the `openldap`-dependent bits of `cyrus-sasl-ldap`.
# The spec was split to break the cyclic build dependency between `openldap` and `cyrus-sasl`.
%define _soversion 3
```